### ISSUE TITLE
[BUG] - Default install fails in privileged LXC Container

### ISSUE THREAD

**Issue by defnull** [ebbba-org/ansible-role-bigbluebutton#223]
**Describe the bug**

Starting with commit e177bc3a863c8145ef018b083e5afb9416398eb1, the freeswitch tasks try to install `auditd` and `apparmor-utils` if `bbb_container_compat` is false. Installing `auditd` in an LXC container fails, even in privileged mode where `bbb_container_compat` should not be required. The commit does not really explain why this package is needed, and it does not seem to be required to change scheduling for freeswitch or html5.

**To Reproduce**
1. Create a privileged LXC Container
2. Try to deploy BBB using default settings.

**Expected behavior**
Installing BBB to a privileged LXC container should not fail with default settings (`bbb_container_compat: false`)

**Comment by defnull** [comment]
Yes, we currently install in `bbb_container_compat ` mode and apply the useful bits of non-compat-mode in our own tasks after the role.

**Comment by BrutalBirdie** [comment]
Sadly I did do not work with LXC Containers so much, do you think you could solve this issue as a contribution?

**Comment by defnull** [comment]
The fix would be to just not install `auditd`. It is not needed to run BBB, just for debugging and people firm enough to analyze auditd logs would also know what to install.

**Comment by defnull** [comment]
Fixed in #380
